src/amber/cli/commands/routes.cr

```diff
@@ -97,15 +96,23 @@ module Amber::CLI
         routes << route
       end
 
+      private def build_route(route_match)
+        build_route(
+          verb: route_match[1]?, controller: route_match[3]?,
+          action: route_match[4]? || "", pipeline: current_pipe,
+          scope: current_scope, uri_pattern: route_match[2]?
```

Comment:
@docelic - This overload allowed for refactoring.

Reply:
Great, didn't think of it. Thanks for letting me know.